tests/test_typeclass/test_typed_dict.py

```diff
@@ -8,6 +8,7 @@
 if sys.version_info[:2] >= (3, 9):  # noqa: C901
```

Comment:
You can remove that now, because there are no python versions that won't execute this branch.